cityofaustin/atd-vz-data
# DB changes to support the fatalities UI

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/17711

## Testing
**URL to test:** 
local

**Steps to test:**
1. Follow steps in https://github.com/cityofaustin/atd-vz-data/pull/1478 to start up the data model and run the cris import.
2. Start your VZE, go to the Fatalities page.
3. Go to the crash details page of the crashes on the fatalities page, edit the people in the crash to add more fatal injuries then go back to the fatalities page to see that reflected. Test adding FRB status and recommendations to the crash, see that in the Fatalities page table. FYI weird bug with recommendations where you have to add a status or partner first before you can add a rec or update, looks like its a preexisting bug but just noticed it with this PR.
4. On the Fatalities page test sorting columns and search by victim name, crash id, etc. _( Also annoying bug with the search where if you type in non ints to an int search field like crash id it bombs, and then because of the way we store search fields in local storage you cant reset the field without clearing your cookies :melting_face: )_
6. Test exporting the rows to .csv using the download button


---
#### Ship list
